docs/user-guide/helm.md

```diff
@@ -1,56 +1,75 @@
-# Installing Ambassador with Helm
+# Installing the Ambassador Edge Stack with Helm
 
-[Helm](https://helm.sh) is a package manager for Kubernetes that automates the release and management of software on Kubernetes. The Ambassador Edge Stack can be installed via a Helm chart with a few simple steps, depending on if you are deploying for the first time, or upgrading from an existing installation.
+[Helm](https://helm.sh) is a package manager for Kubernetes that automates the release and management of software on Kubernetes. The Ambassador Edge Stack can be installed via a Helm chart with a few simple steps, depending on if you are deploying for the first time, upgrading the Ambassador Edge Stack from an existing installation, or migrating from the Ambassador API Gateway .
 
-## Prerequisites
+## Before You Begin
 
-The Ambassador Edge Stack Helm chart is hosted by Datawire and published at `https://www.getambassador.io`. Start by adding this repo to your helm client with:
+The Ambassador Edge Stack Helm chart is hosted by Datawire and published at `https://www.getambassador.io`.
+
+Start by adding this repo to your helm client with the following command:
 
 ```bash
 helm repo add datawire https://www.getambassador.io
 ```
 
-## First Time Installation
+## Install with Helm
+
+When you run the Helm chart, it installs the Ambassador Edge Stack. You can deploy it with either the Helm 2 or Helm 3 chart.
```

Comment:
"You can deploy it with either Helm 2 or Helm 3."

In other words, you can use either version of the tool. There is only one chart.